src/styles/utils/_loops.scss

```
@@ -0,0 +1,28 @@
+@each $hand, $animTime in $hourHands {
+
+  .clock__#{$hand} {
+
+    position: absolute;
```

```suggestion
@each $hand, $animTime in $hourHands {
  .clock__#{$hand} {
    position: absolute;
```

No need for a blank line here.  Just a  suggestion though. 😊